core/libraries/rest_api/calculations/Attendee.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace EventEspresso\core\libraries\rest_api\calculations;
+
+use EE_Attendee;
+use EE_Error;
+use EEM_Attendee;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\libraries\rest_api\calculations\Base as Calculations_Base;
+use EventEspresso\core\libraries\rest_api\controllers\model\Base;
+use InvalidArgumentException;
+use WP_REST_Request;
+
+/**
+ * Class Attendee
+ * adds calculated fields to the  REST API output for the /attendees/ endpoint
+ *
+ * @package EventEspresso\core\libraries\rest_api\calculations
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class Attendee extends Calculations_Base
+{
+
+    /**
+     * @since $VID:$
+     * @param $wpdb_row
+     * @return EE_Attendee|\EE_Base_Class
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    private static function getAttendeeObject($wpdb_row)
+    {
+        $attendee = null;
+        if (is_array($wpdb_row) && isset($wpdb_row['Attendee_CPT.ID']) && absint($wpdb_row['Attendee_CPT.ID'])) {
+            $attendee = EEM_Attendee::instance()->get_one_by_ID($wpdb_row['Attendee_CPT.ID']);
+        }
+        return $attendee;
+    }
+
+
+    /**
+     * @param array           $wpdb_row
+     * @param WP_REST_Request $request
+     * @param Base            $controller
+     * @since $VID:$
+     * @return string
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    public static function userAvatar(array $wpdb_row, WP_REST_Request $request, Base $controller)
+    {
+        $attendee = Attendee::getAttendeeObject($wpdb_row);
+        if (! $attendee instanceof EE_Attendee) {
+            return '';
+        }
+        $avatar = get_avatar_url($attendee->email());
```

Comment:
if we're already needing to dig into the array to get the `Attendee_CPT.ID` entry, we could save a little processing by instead grabbing the `Attendee_Meta.ATT_email` value, and passing that to `get_avatar_url()`, no?